### PR TITLE
Add more hopes of success

### PR DESCRIPTION
**Pull request by whitneywilkes**
It will work. Damnit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Does this work?
  please work
+ please please pleaseee
```
Comment (gaj23) [inline README.md:3]: It worked!

**Review by gaj23 — CHANGES_REQUESTED** [review]
It worked! Could you add more text so we can practice more? Thank you!

**Review by whitneywilkes — COMMENTED** [review]
Comcast Sucks but this is great!

**Review by gaj23 — APPROVED** [review]
Great, thank you!

**Pull request merged**